roles/rsyslog/vars/main.yml

```diff
@@ -51,3 +76,12 @@ rsyslog_weight_map:
   'rulesets': '50'
   'input': '90'
   'inputs': '90'
+
+# X.509 certificates related variables
+# Lookup rsyslog netwrok stream driver
+__rsyslog_pki_map:
```

Comment:
you don't need quotes here - but I see above that the convention in this file is to use quotes on string keys - I guess the quotes are needed for the numbers so yaml will interpret them as strings instead of numbers